### ISSUE TITLE
Why don't you talk about types before mentioning structs?

### ISSUE THREAD

**Issue by gevorgyana** [golang/tour#1067]
Context: https://tour.golang.org/moretypes/2

I think you should mention what `type` keyword does in a section that talks about `struct`s. I understand what both of them mean as a user, but you don't mention what `type` does.

**Comment by ALTree** [comment]
Yeah it appears `type` is indeed used for the 1st time in https://tour.golang.org/moretypes/2 but it's not introduced properly. A comment there would probably be useful.
